pkg/metrics/metrics.go

```diff
@@ -49,7 +49,7 @@ var (
 		[]string{"name"},
 		nil)
 	invalidSecretDesc = prometheus.NewDesc(invalidSecretQuery,
-		"Indicates if the pull secret is functional in the openshift namespace.  If not functional, the reason supplied either points to it missing in the openshift namesapce, or if it is missing credentials for registry.redhat.io when they should exist (1 == not functional, 0 == functional).",
+		"Indicates if the install pull secret is functional for pulling images from registry.redhat.io.  If not functional, the reason supplied either points to it being inaccessible, or if it is missing credentials for registry.redhat.io when they should exist (1 == not functional, 0 == functional).",
```

Comment:
Can we replace "not functional" with "invalid"?

Reply:
part of next push